dap_aria_mapping/getters/openalex.py

```diff
@@ -16,6 +16,60 @@
 from dap_aria_mapping import AI_GENOMICS_BUCKET_NAME
```

Comment:
you can also import the aria-mapping bucket name i.e
```suggestion
from dap_aria_mapping import AI_GENOMICS_BUCKET_NAME, BUCKET_NAME
```